### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by slozier**

### CODE REVIEW

**Comment by pjcozzi** [comment]
Awesome @slozier, thanks for all these doc improvements!

Can you send in a Contributor License Agreement (CLA) so we can review and merge this?  See [CONTRIBUTING.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md) and let us know if you have any questions.

File: Source/Scene/DebugAppearance.js [hunk 1]
```diff
@@ -227,7 +227,7 @@ define([
      *
      * @function
      *
-     * @returns String The full GLSL fragment shader source.
+     * @returns {String< The full GLSL fragment shader source.
```
Comment (pjcozzi) [inline Source/Scene/DebugAppearance.js]: This should be `{String}`

**Comment by pjcozzi** [comment]
Thanks @slozier.  Could you make that one fix please?

File: Source/Scene/DebugAppearance.js (hunk 1 above)
Comment (slozier) [inline Source/Scene/DebugAppearance.js]: Good catch.

**Comment by slozier** [comment]
@pjcozzi
Looks like I will probably have more documentation fixes over the coming days. Should I make another pull request when I have another batch of corrections or should I add them to this one?

**Comment by pjcozzi** [comment]
@slozier A separate pull request will make it easier to review.  Thanks again for these fixes and thanks in advance for the next batch!

**Pull request merged**